com.vrcfury.vrcfury/Editor/VF/Service/MathService.cs

```diff
@@ -250,6 +250,13 @@ public class VFAFloatOrConst {
                 return MakeSetter(to, from.constt);
             }
         }
+
+        public Motion MakeCopierForNegativeValues(VFAFloatOrConst from, VFAFloat to) {
```

Comment:
This changes smoothing from working on `0 thru infinity` to `-1 thru 1` which is a breaking change for users smoothing full controller params.